go.mod

```diff
@@ -1,20 +1,26 @@
 module github.com/lightninglabs/lightning-node-connect
 
 require (
-	github.com/btcsuite/btcd/btcec/v2 v2.2.0
+	github.com/btcsuite/btcd/btcec/v2 v2.2.1
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
 	github.com/go-errors/errors v1.0.1
+	github.com/golang/protobuf v1.5.2
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.5.0
 	github.com/kkdai/bstream v1.0.0
-	github.com/lightninglabs/aperture v0.1.18-beta
+	github.com/lightninglabs/aperture v0.1.18-beta.0.20221005153455-1f35f48b1734
+	github.com/lightninglabs/faraday v0.2.8-alpha.0.20220909105059-fea194ffb084
 	github.com/lightninglabs/lightning-node-connect/hashmailrpc v1.0.2
-	github.com/lightningnetwork/lnd v0.15.0-beta.rc3
+	github.com/lightninglabs/loop v0.20.1-beta.0.20220916122221-9c3010150016
+	github.com/lightninglabs/pool v0.5.8-alpha
+	github.com/lightningnetwork/lnd v0.15.0-beta.rc6.0.20221005111311-2efc70a5c492
```

Comment:
these dependency bumps should be in the previous commit. Else that commit wont be able to compile